MoxiWorks/MoxiWorks.Platform/BrandService.cs

```diff
@@ -44,10 +44,10 @@ public BrandService(IMoxiWorksClient client)
         /// determine what MoxiWorksCompanyID you can use.
         /// </param>
         /// <returns>List of brands associated with the requested Agent or Company</returns>
-        public async Task<Response<BrandResults>> GetBrandsAsync(string moxiWorksCompanyId, string moxiWorksAgentId = null)
+        public async Task<Response<BrandResults>> GetBrandsAsync(string moxiWorksCompanyId, string agentId = null)
         {
             var builder = new UriBuilder($"brands")
-                .AddQueryParameter("moxi_works_agent_id", moxiWorksAgentId)
```

Comment:
Can you only use the "moxi_works_agent_Id"? if so I think it should be moxiworkAgentId for the parameter name.

Reply:
looks like documentation needs to be updated. should allow either agent identifier attribute.